### ISSUE TITLE
An error occured when I tried to do this operation:   `Project is required.`

### ISSUE THREAD

**Issue by ampc** [intuit/benten#9]
Hi, I've installed and configured the bot but when I'm trying to create a new issue it displays the following error.

```An error occured when I tried to do this operation: 
 `Component/s is required.`
```
Chat log 
```
jirabot APP [11:20 AM]
Ok! Give me the `Summary` of the story ?

António Carvalho [11:20 AM]
test

jirabot APP [11:20 AM]
Component/s ? 
1 `DevOps`   2 `Web`   3 `Example` 

António Carvalho [11:20 AM]
1

jirabot APP [11:20 AM]
Product ? 
1 `Web1`   2 `Web2`   

António Carvalho [11:20 AM]
1

jirabot APP [11:20 AM]
I am working on it

bot APP [11:20 AM]
An error occured when I tried to do this operation: 
 `Project is required.`
```
Application log
```
11:50:16.769 [ForkJoinPool.commonPool-worker-1] INFO  c.i.b.channel.slack.SlackChannel - Found a conversation catalyst command for action: action_jira_comment_issue .Entering catalyst conversation.
11:50:16.771 [ForkJoinPool.commonPool-worker-1] INFO  c.i.b.channel.slack.SlackChannel - Handling action action=action_jira_create, user=antonio.carvalho
11:50:16.949 [AsyncHttpClient-2-1] INFO  a.s.rtm.SlackRealTimeMessagingClient - Slack RTM message : {"ok":true,"reply_to":0,"ts":"1542109816.017800","text":"I am working on it \n"}
11:50:17.289 [ForkJoinPool.commonPool-worker-1] ERROR c.i.benten.jira.BentenJiraClient - com.intuit.benten.jira.exceptions.BentenJiraException: Project is required.
java.lang.RuntimeException: com.intuit.benten.jira.exceptions.BentenJiraException: Project is required.
        at com.intuit.benten.jira.http.JiraHttpClient.createIssue(JiraHttpClient.java:207)
        at com.intuit.benten.jira.BentenJiraClient.createIssue(BentenJiraClient.java:161)
        at com.intuit.benten.jira.actionhandlers.JiraCreateIssueActionHandler.handle(JiraCreateIssueActionHandler.java:61)
        at com.intuit.benten.channel.slack.SlackChannel.lambda$null$2(SlackChannel.java:246)
        at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:656)
        at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
        at java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:443)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: com.intuit.benten.jira.exceptions.BentenJiraException: Project is required.
        at com.intuit.benten.jira.http.JiraHttpClient.handleJiraException(JiraHttpClient.java:227)
        at com.intuit.benten.jira.http.JiraHttpClient.createIssue(JiraHttpClient.java:200)
        ... 10 common frames omitted
```

**Comment by ampc** [comment]
Okay, found out that there is a custom field required in my case `cf[12402]` which corresponds to projects name.

**Comment by ampc** [comment]
Any guidance on how to add custom fields to requests and responses? Thank you :smiley_cat:

**Comment by DivakarUngatla** [comment]
I assume you were trying the `create jira story`. The way it works for required fields is BenTen calls Jira API to figure out what are all the required fields and then convert them into a conversation. Do you have another required field called `Project` in you Jira create issue UI ? Can you give provide a screenshot. That will help me fixing this.

**Comment by DivakarUngatla** [comment]
Released a new version 0.1.4 with the fix
please try it out and let me know
you can update the `benten.version` in pom.xml of `benten-starter` 
and that should do it.

**Comment by DivakarUngatla** [comment]
Got a confirmation from @ampc  Antonio that the issue has been resolved in Release 0.1.5  in [support-channel](https://bentenapp.slack.com/messages/CAX0M6B8X/details/). Closing the issue as resolved.
